docs/source/using/profiles.adoc

```diff
@@ -0,0 +1,37 @@
+= Minishift Profiles
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+
+[[create-multi-instances-of-minishift]]
+== Create multiple instances of Minishift
+
+With the help of profile feature you can create multiple instances of Minishift with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
+
+[[manage-multi-instances-of-minishift]]
+== Manage multiple instances of Minishift
+
+When you have multiple profiles you can run Minishift commands  against a specific profile using `--profile` flag.
```

Comment:
double space between `commands   against`

Reply:
Not fixed